tests/test_prepare_java.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from docpipeline.prepare_java import prepare_java_toc
+import unittest
+import shutil
+import filecmp
+import yaml
+import pytest
+
+mock_original_toc = "tests/testdata/mock-java-original-toc.yml"
+mock_original_toc_copy = "tests/testdata/mock-java-original-toc-copy.yml"
+mock_updated_toc = "tests/testdata/mock-java-updated-toc.yml"
+
+
+class PrepareTocTest(unittest.TestCase):
+    def setUp(cls):
```

Comment:
Would you mind switching this to use pytest fixtures to stay consistent with (most) of the other tests in this repo?

You would create a fixture like this:
https://github.com/googleapis/doc-pipeline/blob/eebbd26d1537de0f4fa58b04f5e629f0de8769f7/tests/test_generate.py#L30-L33

Then when `test_foo` functions include a `yaml_dir` argument, that fixture is run for it. You could do the same to copy the TOC file into `tmpdir` and avoid needing a `tearDown`.

Reply:
Yeah I can do it that way